universal-login-react/src/core/services/TopUpProviderSupportService.ts

```diff
@@ -0,0 +1,28 @@
+import {Country} from '../models/Country';
+import {TopUpProvider} from '../models/TopUpProvider';
+
+export class TopUpProviderSupportService {
+  constructor(private countries: Country[]) {
+  }
+
+  checkRampSupport(countryName: string): boolean {
+    return countryName === 'United Kingdom';
```

Comment:
We should use semantic names (e.g. country codes). Ain't gonna work well with i18n.